packages/react/jest.config.js

```diff
@@ -36,6 +36,18 @@ module.exports = {
       // https://github.com/carbon-design-system/carbon-charts/issues/594
       functions: 77,
     },
+    './src/components/MapCard/storyFiles/MapboxExample.jsx': {
```

Comment:
Should these be a wildcard for `storyFiles` if this is the new pattern we're using?

Reply:
I was thinking about that too, but I'm not sure this is a new pattern. It is a pretty unusual component we are delivering here.